doc/overviews/index.md

```diff
@@ -0,0 +1,8 @@
+# Overviews
+
+The Python Visualization Landscape can seem daunting at first. These
+overviews attempt to shine light on common patterns and use cases.
```

Comment:
Overviews still to come, I assume?

Reply:
They are in the side bar. I wasn't sure if links should be linked from the sidebar, from the main page or both.

Reply:
Always both.